src/Common/src/Interop/OSX/Interop.EventStream.cs

```diff
@@ -21,7 +21,7 @@ internal static partial class EventStream
         /// <summary>
         /// This constant specifies that we don't want historical file system events, only new ones
         /// </summary>
-        internal const ulong kFSEventStreamEventIdSinceNow = ulong.MaxValue;
+        internal const ulong kFSEventStreamEventIdSinceNow = 0xFFFFFFFFFFFFFFFF;
```

Comment:
Why this change?  Just curious.  I had to count the number of Fs to verify it was correct.